src/index.html

```diff
@@ -3,11 +3,60 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, initial-scale=1.0,">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--N"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
користуйся модифікатором блоку stars--1/2/3 і nth-child щоб проставити активні зірочки

Reply:
haven't fixed